bellbot_action_server/package.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <package>
   <name>bellbot_action_server</name>
-  <version>0.0.10</version>
+  <version>0.0.11</version>
```

Comment:
Didn't know that these are automatically updated by jen. How to fix it? Shall I manually change them back and commit?